zeus/run/dataloader.py

```diff
@@ -989,22 +944,20 @@ def __iter__(self):
         self.sample_num = 0
         self._log(f"Epoch {self.epoch_num} begin.")
 
-        # Start epoch timer.
-        self.epoch_start_time = time.monotonic()
-
         # Cache the dataloader iterator.
         self.iter = super().__iter__()
 
-        # The power limit of the GPU is only changed by the train dataloader.
-        if self._is_train and self.rank == 0:
-            # The train loader always starts the monitor, and the eval loader kills it.
-            self._start_monitor()
-            # If we're not profiling, use the steady state power limit.
-            # If we are profiling, the power limit will be set in __next__ with warmup.
-            # Power limit result is already loaded in when initializing the train dataloader,
-            # so we just set the power limit directly.
-            if not self._should_profile:
-                self._set_gpu_steady_power_limit()
+        if self.rank == 0:
+            # Push epoch profiling window
```

Comment:
So if I understood correctly, do we now have a separate epoch profiling window for train and eval? Either way, worth writing a comment about.

Reply:
Yes. Separate *epoch* profiling windows for train and eval. During the first *epoch* profiling window for train, smaller windows (for each power limit) overlap.

Will update the two ascii arts.

Reply:
Awesome. Thanks!